README.md

```diff
@@ -42,11 +58,14 @@ dataset/aptos/
 
 `diffusion.py`: Contains code for forward and reverse diffusion.
 
-A class for `weighted_loss` has been implemented which takes `weight` as a parameter that is passed from the diffusion parameters `params["diffusion"]["weight"]`. There are three types of loss: 
+A class for `weighted_loss` has been implemented which takes `weight` as a parameter that is passed from the diffusion parameters `params["diffusion"]["weight"]`. There are three types of loss:
+
 - Unweighted MMD loss (`weight=None`)
 - Weighted MMD loss with the inverse of number of images in each class (`weight=n`)
 - Weighted MMD loss with the square root of the inverse of number of images in each class (`weight=sqrt`)
 
+Weights are calculated currently using total images. There are [1805, 370, 999, 193, 295] images for classes [0, 1, 2, 3, 4] respectively. Thus the weight is currently static and set to `[1/1805, 1/370, 1/999, 1/193, 1/295]`. It can be made dynamic by finding the total number of images for each batch and then setting weights based on number of images of each class in the batch. This is an extension of the project.
```

Comment:
We don't need the comment about dynamic weights. It does not make sense to implement it.

Reply:
I think it makes sense, @Sehmimul said it might be a different approach to applying weights. I think it's worth mentioning here for future work. If we don't want it here, we can have a future work section in README, I can mention it there instead.

Reply:
Can you resolve this?

Reply:
Using dynamic weights here does not make sense. I will resolve it after the last two sentences are removed.

Reply:
@Sehmimul can you review it once you have time?

Reply:
Okay let's add a potential extension section and add that dynamic loss part. The sentence  as it is incorrect as some batches can have no images of a particular label. Sentence, needs to be explained properly.